data/settings/1.20.x/autoscaling.toml

```diff
@@ -0,0 +1,15 @@
+[[docs.ref.should-wait]]
+description = """
+If set to `true`, the node waits until the instance reaches the [`InService` state](https://docs.aws.amazon.com/autoscaling/ec2/userguide/ec2-auto-scaling-lifecycle.html#as-lifecycle-inservice) to join the cluster. 
+"""
+accepted_values = [
+    "`true`",
+    "`false`"
+]
+note = """
+Only set this value to `true` if the instance is currently (or will be) in an auto scaling group. 
+"""
+default = "`false`"
+
+[[docs.ref.should-wait.example]]
+value = "true"
```

Comment:
I think the team had this convention to add newline to the EOF. But I also see these are missing in past versions. Maybe starting from 1.20.x we can start enforcing this?

Reply:
fixed it for these toml files